### PR TITLE
Prevent traffic assigned to locally terminated VLANs from being forwarded

### PR DESCRIPTION
**Pull request by wkz**
## Description

Before this change, in a setup like the following, packets assigned to
VLAN 10 were forwarded between the switch ports, even though the
configuration dictates that they should only be locally terminated.

```sh
ip link add dev br0 up type bridge vlan_filtering 1
ip link set dev swp1 master br0
ip link set dev swp2 master br0
ip link add dev swp1.10 link swp1 up type vlan id 10
ip link add dev swp2.10 link swp2 up type vlan id 10
```
```
swp1.10  br0  swp2.10
      \ /   \ /
      swp1  swp2
```

Therefore, make sure that VLANs that are added to the VTU to terminate
a VLAN upper interface, rather than to offload bridge VLANs, are
marked as policy entries. As the VTU policy of user ports is already
set to TRAP (to ensure proper standalone port operation), this will
cause all packets assigned to these VLANs to properly terminated.

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [x] Bugfix
  - [x] Regression tests
  - [x] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [x] Test changes
  - [x] Checked in changed Readme.adoc (make test-spec)
  - [x] Added new test to group Readme.adoc and yaml file
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [ ] Documentation content changes
  - [ ] ChangeLog updated (for major changes)
- [ ] Other (please describe):

### CODE REVIEW

File: test/case/ietf_interfaces/ietf_interfaces.yaml
Comment (mattiaswal) [inline test/case/ietf_interfaces/ietf_interfaces.yaml:1]: Add to test specification as well.

**Comment by troglobit** [comment]
I've had my fingers in this, but the parts by @wkz look good to me to merge.  So we only need input from @mattiaswal on the total set, and the late `pvid = vid` addition for bridges may need some additional offline discussion as well.

**Review by troglobit — APPROVED** [review]

File: test/case/ietf_interfaces/ietf_interfaces.yaml
Comment (wkz) [reply test/case/ietf_interfaces/ietf_interfaces.yaml:1]: Almost remembered everything! :sweat_smile:  Thanks!

**Review by mattiaswal — APPROVED** [review]
Great work :100:  :rocket:

**Pull request merged**